### ISSUE TITLE
Python 3.x  

### ISSUE THREAD

**Issue by maluta** [crudlio/crudl-example-django#10]
Is there any roadpmap do add Python 3.0 support in this example?

**Comment by sehmaschine** [comment]
not really, because the backend is just an example – and we would like to use an external API anyway (so we might get rid of the python part altogether).
